test/parquet/test_filename_column.test

```diff
@@ -56,3 +56,10 @@ query I
 SELECT my_cool_filename FROM read_parquet('data/parquet-testing/enum.parquet', filename=my_cool_filename) LIMIT 1
 ----
 data/parquet-testing/enum.parquet
+
+query III
+from parquet_bloom_probe('?.parquet', 'a', 1)
```

Comment:
you probably meant to use `'__TEST_DIR__/data/parquet-testing/multi_bloom_*.parquet'`